.github/workflows/release.yml

```diff
@@ -0,0 +1,44 @@
+name: Publish
+
+on:
+  # publish from the Releases page:
+  release:
+    types: [published]
```

Comment:
Oh, I think I understand... Does this workflow file add the sdist/wheel to the existing release?

Reply:
That's right. It wasn't clear to me from [the docs for `softprops/action-gh-release@v1`](https://github.com/marketplace/actions/gh-release) immediately either, but I tried it and double checked by reading its code: since each release goes with a specific tag, it creates the release for that tag or reuses the existing release if it's already there.

And btw I chose `published` over `created` because we had this problem last week in ivadomed where there was a scrap release made by accident because a release got Drafted, the workflow ran, but decided not to finish because it wasn't Published, and then there was no obvious way to rerun it. I think only publishing on Published should avoid that.